WordPress/Classes/Utility/Logging/CocoaLumberjack.swift

```diff
@@ -2,26 +2,26 @@ import Foundation
 import CocoaLumberjackSwift
 
 @inlinable
-public func DDLogVerbose(_ message: @autoclosure () -> Any, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
-    CocoaLumberjackSwift.DDLogVerbose("\(message())", file: file, function: function, line: line)
+public func DDLogVerbose(_ message: @autoclosure () -> DDLogMessageFormat, file: StaticString = #file, function: StaticString = #function, line: UInt = #line) {
```

Comment:
Is there a reason we are redefining these?

Reply:
I think the purpose is making the logging functions available to all files in the project. Otherwise, we'll need to add `import CocoaLumberjackSwift` in every file that logs thing.